core/lib/spree/migrations.rb

```diff
@@ -25,11 +25,11 @@ def initialize(config, engine_name)
     def check
       if File.directory?(app_dir)
         unless missing_migrations.empty?
-          puts "[#{engine_name.capitalize} WARNING] Missing migrations."
+          warn "[#{engine_name.capitalize} WARNING] Missing migrations."
           missing_migrations.each do |migration|
-            puts "[#{engine_name.capitalize} WARNING] #{migration} from #{engine_name} is missing."
+            warn "[#{engine_name.capitalize} WARNING] #{migration} from #{engine_name} is missing."
           end
-          puts "[#{engine_name.capitalize} WARNING] Run `bundle exec rake railties:install:migrations` to get them.\n\n"
+          warn "[#{engine_name.capitalize} WARNING] Run `bundle exec rake railties:install:migrations` to get them.\n\n"
```

Comment:
We could try to add a line to explain that this check can be bypassed with `SOLIDUS_SKIP_MIGRATIONS_CHECK` ENV.